Formula/thriftgo.rb

```diff
@@ -0,0 +1,36 @@
+class Thriftgo < Formula
+  desc "Implementation of thrift compiler in go language with plugin mechanism"
+  homepage "https://github.com/cloudwego/thriftgo"
+  url "https://github.com/cloudwego/thriftgo/archive/refs/tags/v0.2.9.tar.gz"
```

Comment:
this did not get  marked as latest release, https://github.com/cloudwego/thriftgo/releases/tag/v0.2.9

Reply:
ok, I created a issue for this https://github.com/cloudwego/thriftgo/issues/91

Reply:
@chenrui333 hi, I have changed the tag to v0.2.0 as the maintainer of cloudwego/thriftgo wants to make a release for v0.3.0

Reply:
sounds good